src/pages/contributors/devin-nusbaum.adoc

```diff
@@ -0,0 +1,111 @@
+= Devin Nusbaum
+:page-name: Devin Nusbaum
+:page-linkedin:
+:page-twitter: 
+:page-github: dwnusbaum
+:page-email:
+:page-image: avatar/devin-nusbaum.jpeg
+:page-pronouns: He/Him/His
+:page-location: Raleigh, North Carolina, USA
+:page-firstcommit: 2017
+:page-datepublished: 2024-10-22
+:page-featured: true
+:page-intro: Devin is a software engineer based in Raleigh, North Carolina. After graduating from Chapel Hill, Devin spent time at a finance company where he first encountered Jenkins. In 2017, after relocating to Raleigh and joining CloudBees, he contributed to the Jenkins project over the last seven years, starting with smaller bug fixes and eventually moving on to working on Jenkins core. He's especially connected to the Pipeline functionality, dedicating time to refining and enhancing the experience. When he's not deep in Jenkins core, Devin has taken up woodworking and making things with his hands. The results of his work include intricate cutting boards that reflect the time and effort needed to create such lovely pieces.
+
+== What is your background prior to contributing to Jenkins?
+
+I went to school at Chapel Hill in North Carolina and after graduating, my first job was at a finance company.
+During my time there, we were working on electronic trading and I was on their platform for about a year.
+The company was using ANT and everything had to be done manually.
+Creating and running the builds was fully manual, so if you wanted to publish something, you had to do it yourself.
+While I was there, they wanted to switch over to CI to try and ease the workflow people were stuck in.
+They asked me to set up Jenkins and move everything over from ANT to Maven. 
+At the time, I thought Maven was nice and easy to use but wasn't as familiar with Jenkins.
+
+== How long have you been using Jenkins?
+
+My first experience with Jenkins was at the finance company back in 2016, prior to joining CloudBees.
+My boss had asked me to set up Jenkins so that the team could move towards CI for their build processes.
+It can be tricky when getting started with CI/CD because with something like Jenkins, there are so many options to work with, but not necessarily a singular path to go down.
```

Comment:
```suggestion
It can be tricky when getting started with CI/CD because with something like Jenkins, there are so many options to work with, but not necessarily a singular path to follow.
```